README.md

```diff
@@ -4,11 +4,9 @@
 Community participation is crucial to the success of a new project. This is why we are launching *Desmos Primer* ("*Primer*"), a simple cookbook that will help you to get started with Desmos and enjoy a brand new social networking experience. 
 
 ## How the Primer works
-To make things as easy as possible we will gradually publish new challenges that, once completed, will grant you `Desmos Points`. To help newer users to get used to the blockchain space, along with each challenge we will publish a step-by-step completion guide to explain the reason behind some specific operations. 
+To make things as easy as possible we will gradually publish new challenges that, once completed, will grant you `Desmos Tokens`. To help newer users to get used to the blockchain space, along with each challenge we will publish a step-by-step completion guide to explain the reason behind some specific operations. 
 
-Once the primer has finished, no new challenges will be published and all your earned Desmos Points will be converted into `Desmos Token` ("`DT`"), the native staking token of the Desmos Chain, based on a conversion scale that we will publish very soon. 
-
-You can complete the challenges in any order you prefer, but please note that you will only get Desmos Points when you have completed the challenges by the given deadlines.
+You can complete the challenges in any order you prefer, but please note that you will only get Desmos Tokens when you have completed the challenges by the given deadlines.
```

Comment:
You can complete the challenges in any order you prefer, but please note that you will only get Desmos Tokens when you have completed the challenges by the given deadlines.

->

Please note that you will only get Desmos Tokens when you have completed the challenges by the given deadlines and conditions.